src/default-settings.js

```diff
@@ -1,167 +1,177 @@
-    nextSettings.marks[2].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[2].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
+    nextSettings.marks[2].values[nextSettings.ser_col] = ['Yes', 'Y'];
+
     nextSettings.marks[3].per = [nextSettings.id_col, nextSettings.seq_col, 'wc_value'];
-    nextSettings.marks[3].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[3].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
+    nextSettings.marks[3].values = {wc_category: [nextSettings.stdy_col]};
+    nextSettings.marks[3].values[nextSettings.ser_col] = ['Yes', 'Y'];
+
+    nextSettings.legend.order = nextSettings.sev_vals;
+
     nextSettings.color_by = nextSettings.sev_col;
 
     return nextSettings;
 }
 
-export const controlInputs = [ 
-    {label: "Severity", type: "subsetter", value_col: "AESEV", multiple: true},
-    {label: "System Organ Class", type: "subsetter", value_col: "AEBODSYS"},
-    {label: "Subject ID", type: "subsetter", value_col: "USUBJID"},
-    {label: "Related to Treatment", type: "subsetter", value_col: "AEREL"},
-    {label: "Serious?", type: "subsetter", value_col: "AESER"},
-    {label: "Sort Ptcpts", type: "dropdown", option: "y.sort", values: ["earliest", "alphabetical-descending"], require: true}
-];
-
-export function syncControlInputs(preControlInputs, preSettings){
-    var severityControl = preControlInputs.filter(function(d){return d.label=="Severity"})[0];
-    severityControl.value_col = preSettings.sev_col;
-
-    var sOCControl = preControlInputs.filter(function(d){return d.label=="System Organ Class"})[0];
-    sOCControl.value_col = preSettings.soc_col;
-
-    var subjectControl = preControlInputs.filter(function(d){return d.label=="Subject ID"})[0];
-    subjectControl.value_col = preSettings.id_col;
-
-    var relatedControl = preControlInputs.filter(function(d){return d.label=="Related to Treatment"})[0];
-    relatedControl.value_col = preSettings.rel_col;
-
-    var seriousControl = preControlInputs.filter(function(d){return d.label=="Serious?"})[0];
-    seriousControl.value_col = preSettings.ser_col;
-
-    settings.filter_cols.forEach(function(d,i){
-      var thisFilter = {
-        type:"subsetter", 
-        value_col:d, 
-        label:d,
-        multiple:true
-      }
-      var filter_vars = preControlInputs.map(function(d){return d.value_col})
-      if (filter_vars.indexOf(thisFilter.value_col)== -1) {
-        preControlInputs.push(thisFilter);
-      } 
-    })
+export const controlInputs =
```

Comment:
Me too!  Updated to push `filters.value_col` to `detail_cols` as you're doing in _safety-histogram_.